pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
     	<groupId>org.jenkins-ci.plugins</groupId>
     	<artifactId>plugin</artifactId>
-        <version>1.557</version>
+        <version>1.586</version>
```

Comment:
Future plugin development will would available to recent versions unless a backwards-compatible branch is maintained with extra effort (like Subversion plugin's `2.4.x` branch while 2.5 aims at Jenkins 1.577 or so)